firefox/lib/window-listener.js

```diff
@@ -11,7 +11,7 @@ const { Cc, Ci } = require("chrome");
 const wm = Cc["@mozilla.org/appshell/window-mediator;1"].
            getService(Ci.nsIWindowMediator);
 
-const { pagemod, workerAttached } = require("service");
+const { pagemod, workerAttached } = require("./service");
```

Comment:
We want to `require()` the `service.js` file that is in this directory. Without the `./`, Firefox will try to load a module called `"service"` which does not exist.